.travis.yml

```diff
@@ -37,38 +38,45 @@ jobs:
     # a scheduled task to build the weekly dev build on the main branch. The
     # weekly frequency is meant to avoid depleting the Travis CI credits too
     # fast.
-    - python: 3.7
-      os: linux
+    - os: linux
       arch: arm64-graviton2
       dist: focal
-      virt: lxd
+      virt: vm
       group: edge
       if: type = cron or commit_message =~ /\[cd build\]/
       env:
-        - BUILD_WHEEL=true
         - CIBW_BUILD=cp37-manylinux_aarch64
+        - BUILD_WHEEL=true
 
-    - python: 3.8
-      os: linux
+    - os: linux
       arch: arm64-graviton2
       dist: focal
-      virt: lxd
+      virt: vm
       group: edge
       if: type = cron or commit_message =~ /\[cd build\]/
       env:
-        - BUILD_WHEEL=true
         - CIBW_BUILD=cp38-manylinux_aarch64
+        - BUILD_WHEEL=true
```

Comment:
I changed the order so that `CIBW_BUILD=cp38` appears first in the UI so it is easier to see the Python version.